docs/library/array.rst

```diff
@@ -1,8 +1,6 @@
 :mod:`array` -- arrays of numeric data
 ======================================
 
```

Comment:
I'd prefer to have these docs inline rather than in files separate from the implementation. Until then I'd like to call them unsupported because its hard to keep them up to date.

Reply:
I'm reluctant to call them unsupported. To me that means you shouldn't use them, or that we wouldn't fix bugs in them. If you want to say that they may change in the future, that seems more like what you want to imply. But we couldn't write a lot of useful code without `struct`, `array`, `sys`, `os`, etc. in our libraries and use them in Learn Guides, so maybe something milder than "unsupported".